modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -228,89 +215,81 @@ $ aws iam put-role-policy --role-name albo-controller --policy-name perms-policy
 +
 .. Generate the controller's AWS credentials:
 +
-[source,terminal]
+[source,terminal,subs="quotes,verbatim"]
 ----
-$ cat <<EOF > albo-controller-aws-credentials.cfg
+$ cat <EOF> albo-controller-aws-credentials.cfg
 [default]
 sts_regional_endpoints = regional
 role_arn = ${CONTROLLER_ROLE_ARN}
 web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 EOF
 ----
```

Comment:
This file is not needed anymore as we will use just the role in `AWSLoadBalanacerController` resource.

Reply:
Thanks. I removed:

`+
.. Generate the controller's AWS credentials:
+
[source,terminal,subs="quotes,verbatim"]
----
$ cat <EOF> albo-controller-aws-credentials.cfg
[default]
sts_regional_endpoints = regional
role_arn = ${CONTROLLER_ROLE_ARN}
web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
EOF
----`